app/views/layouts/nested_node_resource.html.erb

```diff
@@ -1,23 +1,3 @@
-<% content_for :breadcrumbs do %>
```

Comment:
What's the reason for moving this away from the layout? Looking at both the `_breadcrumbs` partials for Notes and Evidence, it seems to be very similar that they can be put into one file. The advantage of this is that we can reduce the `render partial: 'breadcrumbs'` declarations.

Reply:
I decide to move it away from this layout because Nodes, Notes and Evidence are using the same breadcrumbs initially. I can make it into one file, but it'll be like something below.

```
if controller == 'nodes'
  render 'node' breadcrumb partial
elsif controller == 'notes'
  render 'notes' breadcrumb partial
else
  render 'evidence' breadcrumb partial
end
```

Reply:
Adding the following to this file seems to work:
```
<% content_for :breadcrumb do %>
  <%= render 'breadcrumbs' %>
<% end %>
```
without adding a bunch of elsifs. Can you confirm? 

Also, we can move the `render 'nodes/breadcrumb_dropdown'` here too.

Reply:
Thanks @aapomm , it works like a charm!